section3/reflection.md

```diff
@@ -2,12 +2,24 @@
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
 
+I'm still feeling very good about all of this information.  I didn't have to use many outside sources to figure out how to write these lines of code.  I did have to look back at the section 2 exercises to remember the exact syntax of the if statements, but I think that those will become more engrained in my memory after some more practice writing them.
```

Comment:
Yep, the stuff you use regularly will absolutely become more engrained - you seem to be  in a great place.